man/veritytab.xml

```diff
@@ -0,0 +1,167 @@
+    The first four fields are mandatory, the remaining one is optional.</para>
+
+    <para>The first field contains the name of the resulting verity volume; its block device is set up
+    below <filename>/dev/mapper/</filename>.</para>
+
+    <para>The second field contains a path to the underlying block data device, or a specification of a block device via
+    <literal>UUID=</literal> followed by the UUID.</para>
+
+    <para>The third field contains a path to the underlying block hash device, or a specification of a block device via
+    <literal>UUID=</literal> followed by the UUID.</para>
+
+    <para>The fourth field is the <literal>roothash</literal> in hexadecimal.</para>
+
+    <para>The fifth field, if present, is a comma-delimited list of options. The following options are
+    recognized:</para>
+
+    <variablelist class='fstab-options'>
+
+      <varlistentry>
+        <term><option>ignore-corruption</option></term>
+        <term><option>restart-on-corruption</option></term>
+        <term><option>panic-on-corruption</option></term>
+
+        <listitem><para>Defines what to do if data integrity problem is detected (data corruption). Without these
+        options kernel fails the IO operation with I/O error. With <literal>--ignore-corruption</literal> option the
+        corruption is only logged. With <literal>--restart-on-corruption</literal> or
+        <literal>--panic-on-corruption</literal> the kernel is restarted (panicked) immediately.
+
+        (You have to provide way how to avoid restart loops.)</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>ignore-zero-blocks</option></term>
+
+        <listitem><para>Instruct kernel to not verify blocks that are expected to contain zeroes and always directly
+        return zeroes instead.
+
+        WARNING: Use this option only in very specific cases. This option is available since Linux kernel version 4.5.
+        </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>check-at-most-once</option></term>
+
+        <listitem><para>Instruct kernel to verify blocks only the first time they are read from the data device, rather
+        than every time.
+
+        WARNING: It provides a reduced level of security because only offline tampering of the data device's content
+        will be detected, not online tampering. This option is available since Linux kernel version 4.17.
+        </para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>noauto</option></term>
+
+        <listitem><para>This device will not be added to <filename>cryptsetup-pre.target</filename>.
+        This means that it will not be automatically unlocked on boot, unless something else pulls
+        it in. In particular, if the device is used for a mount point, it'll be unlocked
```

Comment:
I'm not sure "unlocked" is the best word here? It's not a crypto device after all. "enabled" maybe?

Reply:
Oops, I forgot to reword that part. I took it from [crypttab(5)](https://www.freedesktop.org/software/systemd/man/crypttab.html).

Reply:
"enabled" seems fine.